doc/introduction/configuration.rst

```diff
@@ -40,20 +40,25 @@ and has the following format:
 
 .. code-block:: toml
 
+    # This is an example config.toml file, the options shown here appear as comments.
+    # Every line starting with a # character is a comment in TOML.
+    # For any of the options to take effect, uncomment the line of the option by
+    # removing the # character and modifying the value.
```

Comment:
Nitpick: For the example config in the documentation, it should be safe to show an uncommented version, right? We could just set `hbar = 2` below

Reply:
(as I imagine, for the version in the documentation to become active, a user would need to explicitly copy it into their directory, so wouldn't be 'surprised' when the configuration takes effect.)